### PR TITLE
fix(@angular/cli): support the sublevel names of projects

### LINKED ISSUE #14797: Scoped library highlighting issue
Hi! I create scoped library with:
`ng g library @bearui/angular`
and in IDEA I see:
image
with Property `@bearui/angular` is not allowed
Is this correct highlighting?

### PR DESCRIPTION
**Pull request by Eusen**
change projects.patternProperties to npm.name.pattern

Closes #14797

### CODE REVIEW

File: packages/angular/cli/lib/config/schema.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "projects": {
       "type": "object",
       "patternProperties": {
-        "^[a-zA-Z][.0-9a-zA-Z]*(-[.0-9a-zA-Z]*)*$": {
+        "^(?:@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*$": {
```
Comment (alan-agius4) [inline packages/angular/cli/lib/config/schema.json]: The `/` should probably be escaped

```
^(?:@[a-z0-9-~][a-z0-9-._~]*\/)?[a-z0-9-~][a-z0-9-._~]*$
```

Also can you add the below footer to your commit

```

Closes #14797
```

Thanks and sorry for the delay.

**Comment by alan-agius4** [comment]
Ps: might also require a rebase

**Comment by alan-agius4** [comment]
Hi @Eusen, are you willing to address the comments please?

File: packages/angular/cli/lib/config/schema.json (hunk 1 above)
Comment (Eusen) [reply packages/angular/cli/lib/config/schema.json]: Never mind. How long will Lvy be released? I'm looking forward to it.

**Pull request merged**